packages/utils/formatNumber.ts

```diff
@@ -1,8 +1,8 @@
 import BigNumber from 'bignumber.js'
 
-const ZERO = new BigNumber(0)
-const ONE = new BigNumber(1)
-const TEN = new BigNumber(10)
+export const ZERO = new BigNumber(0)
+export const ONE = new BigNumber(1)
+export const TEN = new BigNumber(10)
```

Comment:
please remove all of these, use from './bigNumber.ts'

Reply:
done